### PR TITLE
test: mark unit test secrets as not secret

### PR DESCRIPTION
**Pull request by BlaineEXE**
There are 2 cases of randomly generated secrets copied into Rook's unit test code that have been flagged by Gitleaks. Add a comment to both cases to help the tool understand that these aren't real production secrets -- just unit test stand-ins.

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by subhamkrai — COMMENTED** [review]
Where we trigger `Gitleaks` tool or it is just your local thing?

File: pkg/operator/ceph/object/controller_test.go [hunk 1]
```diff
@@ -348,7 +348,7 @@ var mockMultisiteAdminOpsCtxFunc = func(objContext *Context, spec *cephv1.Object
 	return &AdminOpsContext{
 		Context:               *context,
 		AdminOpsUserAccessKey: "EOE7FYCNOBZJ5VFV909G",
-		AdminOpsUserSecretKey: "qmIqpWm8HxCzmynCrD6U6vKWi4hnDBndOnmxXNsV",
+		AdminOpsUserSecretKey: "qmIqpWm8HxCzmynCrD6U6vKWi4hnDBndOnmxXNsV", // notsecret
```
Comment (parth-gr) [inline pkg/operator/ceph/object/controller_test.go:351]: // not production secret

Comment (BlaineEXE) [reply pkg/operator/ceph/object/controller_test.go:351]: The specific instructions I received in the email from gitleaks were that this must be `// notsecret`. It's a directive to a program, not a note to a person.

**Comment by BlaineEXE** [comment]
> Where we trigger `Gitleaks` tool or it is just your local thing?

I received an email from corporate IT about this in my BlaineEXE fork of Rook, and it seemed simple enough to fix.

**Review by travisn — APPROVED** [review]

**Pull request merged**